platform/ethereum/domain.go

```diff
@@ -1,14 +1,34 @@
 package ethereum
 
 import (
+	"strings"
+
 	CoinType "github.com/trustwallet/blockatlas/coin"
+	"github.com/trustwallet/blockatlas/pkg/address"
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
 	"github.com/trustwallet/blockatlas/pkg/errors"
 	"github.com/trustwallet/blockatlas/pkg/logger"
 	"github.com/trustwallet/blockatlas/platform/ethereum/ens"
 	AddressEncoder "github.com/trustwallet/ens-coincodec"
```

Comment:
Remove naming of packages please

Reply:
You mean aliases like CoinType?

Reply:
yep, cannot see any sense to do it

Reply:
https://github.com/uber-go/guide/blob/master/style.md#import-aliasing

Reply:
OK, I agree.  (Sorry, first I thought your comment refers to the changed line.). Done